custom_components/elegoo_printer/camera.py

```
@@ -0,0 +1,93 @@
+        self._attr_unique_id = coordinator.generate_unique_id(
+            self.entity_description.key
+        )
+
+        runtime_data = coordinator.config_entry.runtime_data
+        if not runtime_data or not runtime_data.client:
+            raise PlatformNotReady("Printer client not yet available")
+        self.printer_client: ElegooPrinterClient = runtime_data.client._elegoo_printer
+        if not self.printer_client.ip_address:
+            raise PlatformNotReady("Printer IP address not available")
+
+        ## Temporary until we finish testing
```

Flake8 (7.2.0)

[error] too many leading '#' for block comment

(E266)